.release-please-manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "packages/api-explorer": "21.0.10",
+  "packages/extension-api-explorer": "21.0.10",
+  "packages/extension-sdk": "21.0.10",
+  "packages/extension-sdk-react": "21.0.10",
+  "packages/hackathon": "21.0.10",
+  "packages/run-it": "21.0.10",
+  "packages/sdk": "21.0.10",
+  "packages/sdk-codegen": "21.0.10",
+  "packages/sdk-codegen-scripts": "21.0.10",
+  "packages/sdk-codegen-utils": "21.0.10",
+  "packages/sdk-node": "21.0.10",
+  "packages/sdk-rtl": "21.0.10",
+  "packages/wholly-sheet": "21.0.10",
```

Comment:
this is the current version for all these packages. release-please will suggest bumps starting from here. Let's wait to see the release PR and then we can come back and fudge some of these "initial" versions to get something we actually want where applicable